package.json

```diff
@@ -16,19 +16,28 @@
   },
   "xo": {
     "parser": "babel-eslint",
-    "extends": ["prettier", "prettier/react", "plugin:react/recommended"],
-    "env": ["browser", "node"],
+    "extends": [
+      "prettier",
+      "prettier/react",
+      "plugin:react/recommended"
+    ],
+    "env": [
+      "browser",
+      "node"
+    ],
     "rules": {
       "linebreak-style": 0,
       "react/display-name": 0,
       "react/prop-types": 0
     },
     "space:": 2,
-    "ignores": ["next.config.js"]
+    "ignores": [
+      "next.config.js"
+    ]
   },
   "lint-staged": {
     "*.js": [
-      "prettier --write --single-quote --print-width=80 --trailing-comma=all",
+      "prettier --write --single-quote --print-width=80 --trailing-comma=all --no-semi",
```

Comment:
Why did you disable semicolons?

Reply:
more of a coding style and also one less thing to worry about

Reply:
If you don't like semicolons then you don't type them in code, let tools like prettier and xo place them at the time of committing(which will be taken care by pre-commit hook `lint-staged`).